LayoutTests/fast/canvas/canvas-intrect-size-overflow.html

```diff
@@ -0,0 +1,45 @@
+<head>
```

Comment:
intrect isn't a HTML/DOM concept.
Maybe you could call it "canvas-imagedata-dimensions-no-overflow.html" or something

Reply:
Fixed